server/datastore/mysql/apple_mdm.go

```diff
@@ -942,7 +940,8 @@ func (ds *Datastore) ListMDMAppleProfilesToInstall(ctx context.Context) ([]*flee
             SELECT
               macp.profile_id,
               h.uuid as host_uuid,
-              macp.identifier as profile_identifier
+              macp.identifier as profile_identifier,
+			  macp.name as profile_name
```

Comment:
Nit re: indentation.

Reply:
Hmm I think the `profile_name` is missing from the top-level `SELECT` and would be unused as it stands.